Dockerfile

```diff
@@ -1,26 +1,24 @@
+ARG PIP_VERSION=22.0.2
+
 FROM ubuntu:20.04 as build-image
 
-ARG http_proxy
-ARG https_proxy
-ARG no_proxy="nuclio,${no_proxy}"
-ARG socks_proxy
```

Comment:
Why proxy settings were removed?

Reply:
`http_proxy`, `https_proxy` and `no_proxy` are all [predefined](https://docs.docker.com/engine/reference/builder/#predefined-args) in Docker, while `socks_proxy` is not a standard variable and won't do anything unless it's used explicitly - which, in the build stage, it isn't.